conf/resources/api12_schema_1.1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "title": "PushNotificationRequest",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Schema for AMLS ETMP push notification request (04AY) - v7.0",
+  "type": "object",
+  "required": [
+    "name",
+    "email",
+    "variation",
+    "safeId"
+  ],
+  "definitions": {
+    "name": {
+      "type": "string",
+      "pattern": "^.{1,140}$",
```

Comment:
This is the only line that has changed, to relax the validation on the 'name' field.